examples/tutorial/lessons/lesson-02-first-alert.adoc

```diff
@@ -0,0 +1,257 @@
+
+== Lesson 02 - First Alert
+
+We define a _Trigger_ in Hawkular Alerting (hAlerting) to fire an _Alert_ when its condition set evaluates to true.  Let's start with the most simple trigger and build it up with a few common options...
+
+----
+Trigger
+  id:   lesson-02
+  name: Lesson 02
+----
+
+* hAlerting supports multi-tenancy, so everything is assigned to a tenant.
+** The tenantId will be assigned in the REST request, via an HTTP header.
+** In the tutorial we'll put everything in the _tutorial_ tenant.
+* TriggerIds must be unique within the tenant.
+* Name is required, but used for display.
+
+The trigger above uses defaults for all options and it does not yet define any conditions. Using your preferred mechanism (curl, Chrome ARC, etc), let's create the trigger via the REST API:
+
+[cols="1,5l"]
+|===
+|Request
+|POST http://localhost:8080/hawkular/alerts/triggers
+
+|HTTP Headers
+|Hawkular-Tenant: tutorial
+Content-Type: application/json
+
+|Body
+|
+{
+"id":"lesson-02",
+"name":"Lesson 02"
+}
+|===
+
+The request response returns the new trigger.  But just to become more familiar with the REST API, let's fetch the trigger definition and let's look at some of the defaults and missing information:
+
+[cols="1,5l"]
+|===
+|Request
+|GET http://localhost:8080/hawkular/alerts/triggers/lesson-02
+
+|HTTP Headers
+|Hawkular-Tenant: tutorial
+Content-Type: application/json
+|===
+
+.Response
+----
+{
+"tenantId": "tutorial",
+"id": "lesson-02",
+"name": "Lesson 02",
+"type": "STANDARD",
+"eventType": "ALERT",
+"eventCategory": null,
+"eventText": null,
+"severity": "MEDIUM",
+"autoDisable": false,
+"autoEnable": false,
+"autoResolve": false,
+"autoResolveAlerts": true,
+"autoResolveMatch": "ALL",
+"enabled": false,
+"firingMatch": "ALL",
+"source": "_none_"
+}
+----
+
+We'll ignore some fields for now, but a couple of settings are important:
+
+* Severity
+** HAlerts has LOW, MEDIUM, HIGH for alert severity
+* AutoDisable
+** If true the trigger disables after it fires, this can prevent alert floods.
+** We'll turn this on a little later...
+* AutoEnable
+** If true the trigger enables itself when it no longer has unresolved alerts.
+** This avoids having to remember to re-enable a trigger after an issue is resolved.
+* Enabled
+** The trigger won't fire unless it's enabled.
+** We'll set this to true when we have our trigger fully defined.
+
+=== Conditions
+
+The big thing we're missing are the conditions that define when the trigger should fire an alert.  HAlerts offers http://www.hawkular.org/community/docs/developer-guide/alerts.html#\_conditions[many different condition types] out of the box.  Each condition performs evaluations against incoming data to see if it is satisfied.  A condition type found in virtually any alerting system is _Threshold_, so let's start with a threshold condition for our tutorial. Note that the body supports a list of conditions but for now we'll only use one.
```

Comment:
Minor: _Threshold_, is not rendered in italic font. On my editor double "__" did the trick.